src/main/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountsRestClient.java

```diff
@@ -81,7 +81,7 @@ public SuggestAccountsRequest suggestAccounts(String query) throws RestApiExcept
     }
 
     private List<AccountInfo> getSuggestAccounts(String queryPart) throws RestApiException {
-        String request = String.format("/accounts/?%s", queryPart);
+        String request = String.format("/accounts/?suggest&%s", queryPart);
```

Comment:
Is this change required?

Reply:
with param "suggest", detail information like name etc. will be included in the query result. or only account id is returned.